pkg/report/report.go

```diff
@@ -1021,8 +1032,8 @@ func (r *pdfReporter) createRiskMitigationStatus(parsedModel *types.Model, tempF
 	stackedBarChartRiskTracking := chart.StackedBarChart{
 		Width: 4000,
 		//Height: 2500,
-		XAxis: chart.Style{Show: false, FontSize: 26, TextVerticalAlign: chart.TextVerticalAlignBottom},
-		YAxis: chart.Style{Show: true, FontSize: 26, TextVerticalAlign: chart.TextVerticalAlignBottom},
+		XAxis: chart.Style{Hidden: true, FontStyle: chart.FontStyle{FontSize: 26}, TextVerticalAlign: chart.TextVerticalAlignBottom},
```

Comment:
The `incompatible` API required `Show` to be set, but the 2.0 released version instead has a `Hidden` boolean so that elements are default enabled.  We forked from this 2.0 API with the `Hidden` field.